ios/RCTOneSignal/RCTOneSignalEventEmitter.m

```diff
@@ -13,6 +13,8 @@
 
 @implementation RCTOneSignalEventEmitter {
     BOOL hasListeners;
+    NSMutableDictionary* notificationCompletionCache;
+    NSMutableDictionary* receivedNotificationCache;
```

Comment:
In general instance variables like these should start with an underscore in objc so notificationCompletionCache -> _notificationCompletionCache. (Same applies to hasListeners)